src/tests/utils/database-helpers.ts

```diff
@@ -229,6 +239,31 @@ export default class DatabaseHelpers {
     return String(addedToolData.id);
   }
 
+  /**
+   * Inserts note visit mock into db
+   *
+   * @param visit object which contain all info about noteVisit (visitedAt is optional)
+   *
+   * if no visitedAt passed, then visited_at would have CURRENT_DATE value
+   */
+  public async insertNoteVisit(visit: NoteVisitCreationAttributes): Promise<NoteVisit> {
+    const visitedAt = visit.visitedAt ?? 'NOW()';
+
+    // eslint-disable-next-line
```

Comment:
Why this line here? I think it's best to get rid of it

Reply:
this is eslint syntax if i remove it, linter will raise warning on next line

Reply:
So something's wrong - it shouldn't be warn

Reply:
we use it for better readability, because orm insert query returns array [result, metadata], if we will just get results as query[ 0 ] it would be hard to understand, so we decided to use destructor, but metadata after that is never used, this is why linter raises warning

Reply:
Oh, okay! But I think comment can be useful in this place